packages/discovery-react-components/src/components/SearchInput/SearchInput.tsx

```diff
@@ -72,7 +78,7 @@ const SearchInput: FC<SearchInputProps> = ({
   id,
   splitSearchQuerySelector = ' ' as string,
   completionsCount = 5,
-  showAutocomplete = true,
+  showAutocomplete,
```

Comment:
Defaulting to `true` here appears to be wrong, at least according to the tests in `SearchInput.test.tsx` -- if this property is not set, then it should default to value from `componentSettings` (line 103 below).